File: 0162-find-peak-element/0162-find-peak-element.py
class Solution(object):
    def findPeakElement(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        idx=0
        mini = nums[0]
        for i in range(len(nums)):
            if nums[i]>mini:
                mini=nums[i]
                idx=i
        return idx

File: 0162-find-peak-element/0162-find-peak-element.java
class Solution {
    public int findPeakElement(int[] nums) {
        int n = nums.length;
        int maxi = nums[0];
        int idx=0;
        for(int i=0; i<n; i++){
            if(maxi<nums[i]){
                maxi=Math.max(maxi, nums[i]);
                idx=i;
            }

        }
        return idx;
        
    }
}